.husky/pre-commit

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-npm run lint
+npx lint-staged
```

Comment:
I prefer not to use staged as I don't stage my files

Reply:
you should stage your files 😁

Reply:
I select them in my commit command instead - same result